### ISSUE TITLE
GetLogicalProcessorInformationEx fails

### ISSUE THREAD

**Issue by martty** [wolfpld/tracy#871]
Hi, 

Commit 8f3b312 introduced some CPU topology code that fails on my machine, more specifically 
 _GetLogicalProcessorInformationEx with RelationProcessorDie gives an error of 31 (ERROR_GEN_FAILURE). 

https://github.com/wolfpld/tracy/blob/8f3b312b7caf2e974e2879fd063e48f6bf06b6ac/public/client/TracyProfiler.cpp#L3750

Unsure what this means, but the assert triggers afterwards.

My system is
OS Name	Microsoft Windows 10 Home
Version	10.0.19045 Build 19045
Processor	AMD Ryzen 9 3900X 12-Core Processor, 3793 Mhz, 12 Core(s), 24 Logical Processor(s)

**Comment by wolfpld** [comment]
What's the value of `res` when assert hits?

**Comment by martty** [comment]
0

**Comment by wolfpld** [comment]
Can you check what `GetLastError()` returns after the failed call?

**Comment by martty** [comment]
That was 31, what I referred to in the first message.

**Comment by cod3monk3y** [comment]
Just started using Tracy today, and ran into this problem. 

Windows 10 Enterprise
Version: 22H2
OS build 19045.4780

Tracy commit e26c3434

CMake, installed using FetchContent as recommended in the docs.

```
FetchContent_Declare (
	tracy
	GIT_REPOSITORY https://github.com/wolfpld/tracy.git
	GIT_TAG master
	GIT_SHALLOW TRUE
	GIT_PROGRESS TRUE 
)
FetchContent_MakeAvailable (tracy)
```

**Comment by cod3monk3y** [comment]
Reverting to `GIT_TAG "v0.11.1"` fixes the issue. As OP mentioned, looks like `8f3b312b Get CPU die information on windows.` commit caused the problem for me too.

**Comment by wolfpld** [comment]
Should be fixed with 8e3d2506.

**Comment by martty** [comment]
Confirm, fixed for me. Thank you!
